src/Model/User.php

```diff
@@ -395,7 +395,22 @@ public static function getFirstAdmin(array $fields = [])
 	public static function getOwnerDataById(int $uid, bool $repairMissing = true)
 	{
 		if ($uid == 0) {
-			return self::getSystemAccount();
+			$system = self::getSystemAccount();
+
+			$system['publish'] = false;
```

Comment:
I think this should be added directly in `getSystemAccount`.

Reply:
I assumed that `getSystemAccount` was meant to return just a usr record.

Reply:
When looking at where it is called, it is more a replacement for the function `getOwnerDataById` - which returns an `owner-view` record.

But the other thing I just realized is that the owner array of the system user shouldn't be used for this frontend stuff at all.

Reply:
Yeah, it doesn't make sense to call `Profile::load` with `$uid = 0`, an Exception would help troubleshoot it. The system user has no business having a profile.

Reply:
Well, there is a single exception: The HCard is needed for the Diaspora communication. And there `Profile:load` is used.